src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj

```diff
@@ -15,6 +15,16 @@
     <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
This leads me to believe that this project is being treated as a data plane project now. If that is the case we need to fix it.

Reply:
It is inclded in the build by ```Microsoft.Azure.EventGrid.Tests.csproj``` so yes I guess it is.

Reply:
Just because it is included in the Test project doesn't make it a data-plane project. If these properties are necessary then I suspect this project is being detected as a data-plane but it should still be a management. Please double check whether or not these are necessary and if they are please let me know why.

Reply:
Turns out they weren't needed.